### ISSUE TITLE
Latest version causes svg to disappear

### ISSUE THREAD

**Issue by chamkank** [svg/svgo#832]
I recently upgraded my svgo version and it seems that the latest version (1.0.2) outputs a blank svg for a specific svg file. Not sure if this is an issue with the svg file (which worked with at least one previous version of svgo), or an issue introduced with the newer releases. Anyway, here's the original and the compressed version:

**Original svg**
```
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!-- Generator: Adobe Illustrator 16.0.4, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->

<svg
   xmlns:dc="http://purl.org/dc/elements/1.1/"
   xmlns:cc="http://creativecommons.org/ns#"
   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
   xmlns:svg="http://www.w3.org/2000/svg"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   version="1.1"
   id="Layer_1"
   x="0px"
   y="0px"
   width="865px"
   height="146px"
   viewBox="-0.154 168.199 865 146"
   enable-background="new -0.154 168.199 865 146"
   xml:space="preserve"
   sodipodi:docname="red-transparent.svg"
   inkscape:version="0.92.0 r"><metadata
     id="metadata37"><rdf:RDF><cc:Work
         rdf:about=""><dc:format>image/svg+xml</dc:format><dc:type
           rdf:resource="http://purl.org/dc/dcmitype/StillImage" /><dc:title></dc:title></cc:Work></rdf:RDF></metadata><defs
     id="defs35" /><sodipodi:namedview
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1"
     objecttolerance="10"
     gridtolerance="10"
     guidetolerance="10"
     inkscape:pageopacity="0"
     inkscape:pageshadow="2"
     inkscape:window-width="958"
     inkscape:window-height="1057"
     id="namedview33"
     showgrid="false"
     inkscape:zoom="0.82727406"
     inkscape:cx="252.12069"
     inkscape:cy="146.10604"
     inkscape:window-x="960"
     inkscape:window-y="21"
     inkscape:window-maximized="0"
     inkscape:current-layer="Layer_1" /><path
     style="fill:#e2231a"
     d="M 0 0.28125 L 0 145.71875 L 288.10352 145.71875 L 288.10352 0.28125 L 0 0.28125 z M 36.013672 36.294922 L 69.255859 36.294922 C 78.258859 36.294922 85.877609 39.065109 90.724609 43.912109 C 94.880609 48.067109 96.957031 53.607203 96.957031 60.533203 C 96.957031 72.306203 90.723703 79.925672 81.720703 83.388672 L 99.726562 109.70508 L 81.029297 109.70508 L 65.792969 86.160156 L 51.941406 86.160156 L 51.941406 109.70703 L 36.013672 109.70703 L 36.013672 36.294922 z M 113.58008 36.294922 L 168.29102 36.294922 L 168.29102 50.837891 L 129.50781 50.837891 L 129.50781 65.382812 L 163.44141 65.382812 L 163.44141 79.927734 L 129.50781 79.927734 L 129.50781 95.162109 L 168.98242 95.162109 L 168.98242 109.70703 L 113.57812 109.70703 L 113.58008 36.294922 z M 185.60352 36.294922 L 213.99805 36.294922 C 236.85205 36.294922 252.78125 52.224594 252.78125 72.308594 C 252.78125 93.086594 236.85205 109.01367 213.99805 109.01367 L 185.60352 109.01367 L 185.60352 36.294922 z M 51.941406 50.837891 L 51.941406 71.615234 L 67.869141 71.615234 C 75.487141 71.615234 80.335938 67.460562 80.335938 61.226562 C 80.335938 54.301562 75.487141 50.837891 67.869141 50.837891 L 51.941406 50.837891 z M 201.5332 50.837891 L 201.5332 94.470703 L 213.99805 94.470703 C 227.15705 94.470703 236.16016 85.467 236.16016 73 C 236.16016 59.841 227.15705 50.837891 213.99805 50.837891 L 201.5332 50.837891 z "
     id="rect2"
     transform="translate(-0.154,168.199)" /><g
     id="g28"><path
       style="fill:#808184"
       inkscape:connector-curvature="0"
       id="path14"
       d="m 355.817,204.494 h 11.775 l 31.858,72.719 h -13.159 l -7.617,-17.315 h -33.937 l -7.618,17.315 h -13.158 z m 18.008,44.324 -12.464,-29.087 -12.466,29.087 z" /><path
       style="fill:#808184"
       inkscape:connector-curvature="0"
       id="path16"
       d="m 403.605,241.892 v 0 c 0,-20.776 15.236,-37.397 37.398,-37.397 13.157,0 21.469,4.849 28.393,11.08 l -8.31,9.696 c -5.54,-5.541 -11.774,-9.003 -20.083,-9.003 -13.853,0 -23.547,11.08 -23.547,25.624 v 0 c 0,13.853 9.696,25.625 23.547,25.625 9.003,0 14.543,-3.461 20.775,-9.004 l 8.311,8.312 c -7.618,7.617 -15.929,12.465 -29.087,12.465 -21.468,-0.692 -37.397,-16.62 -37.397,-37.398 z" /><path
       style="fill:#808184"
       inkscape:connector-curvature="0"
       id="path18"
       d="m 508.182,204.494 h 11.773 l 31.857,72.719 h -13.158 l -7.618,-17.315 H 497.1 l -7.618,17.315 h -13.158 z m 18.005,44.324 -12.466,-29.087 -12.466,29.087 z" /><path
       style="fill:#808184"
       inkscape:connector-curvature="0"
       id="path20"
       d="m 563.584,205.188 h 27.012 c 22.854,0 38.088,15.235 38.088,36.013 v 0 c 0,20.085 -15.928,36.013 -38.088,36.013 h -27.012 z m 13.16,11.772 v 49.173 h 13.852 c 15.235,0 24.932,-10.389 24.932,-24.241 v 0 c 0,-13.851 -9.696,-24.238 -24.932,-24.238 h -13.852 z" /><path
       style="fill:#808184"
       inkscape:connector-curvature="0"
       id="path22"
       d="m 643.921,205.188 h 53.327 v 11.772 h -40.859 v 18.699 H 692.4 v 11.082 h -36.012 v 19.392 h 41.553 v 11.08 H 643.92 v -72.025 z" /><path
       style="fill:#808184"
       inkscape:connector-curvature="0"
       id="path24"
       d="m 713.177,205.188 h 13.158 l 22.162,33.935 22.161,-33.935 h 13.158 v 72.025 H 771.351 V 225.964 L 748.497,259.9 h -0.692 L 724.95,225.963 v 51.249 h -11.772 v -72.024 z" /><path
       style="fill:#808184"
       inkscape:connector-curvature="0"
       id="path26"
       d="m 823.293,248.818 -28.395,-43.631 h 15.235 l 20.083,31.857 20.085,-31.857 h 14.544 l -28.394,43.631 v 29.088 h -12.467 v -29.088 z" /></g></svg>
```

**Compressed by svgo**
```
<svg xmlns="http://www.w3.org/2000/svg" width="865" height="146"><path d="M-.154 168.48v145.438H287.95V168.48H-.154zm36.014 36.014h33.242c9.003 0 16.622 2.77 21.469 7.617 4.156 4.155 6.232 9.695 6.232 16.621 0 11.773-6.233 19.393-15.236 22.856l18.006 26.316H80.875L65.64 254.36H51.787v23.547H35.86v-73.412zm77.566 0h54.711v14.543h-38.783v14.545h33.933v14.545h-33.933v15.234h39.474v14.545h-55.404l.002-73.412zm72.024 0h28.394c22.854 0 38.783 15.93 38.783 36.014 0 20.778-15.929 36.705-38.783 36.705H185.45v-72.72zM51.787 219.037v20.777h15.928c7.618 0 12.467-4.154 12.467-10.388 0-6.925-4.849-10.39-12.467-10.39H51.787zm149.592 0v43.633h12.465c13.16 0 22.162-9.004 22.162-21.471 0-13.159-9.003-22.162-22.162-22.162H201.38z" fill="#e2231a"/><g fill="#808184"><path d="M355.817 204.494h11.775l31.858 72.719h-13.159l-7.617-17.315h-33.937l-7.618 17.315h-13.158zm18.008 44.324l-12.464-29.087-12.466 29.087zM403.605 241.892c0-20.776 15.236-37.397 37.398-37.397 13.157 0 21.469 4.849 28.393 11.08l-8.31 9.696c-5.54-5.541-11.774-9.003-20.083-9.003-13.853 0-23.547 11.08-23.547 25.624 0 13.853 9.696 25.625 23.547 25.625 9.003 0 14.543-3.461 20.775-9.004l8.311 8.312c-7.618 7.617-15.929 12.465-29.087 12.465-21.468-.692-37.397-16.62-37.397-37.398zM508.182 204.494h11.773l31.857 72.719h-13.158l-7.618-17.315H497.1l-7.618 17.315h-13.158zm18.005 44.324l-12.466-29.087-12.466 29.087zM563.584 205.188h27.012c22.854 0 38.088 15.235 38.088 36.013 0 20.085-15.928 36.013-38.088 36.013h-27.012zm13.16 11.772v49.173h13.852c15.235 0 24.932-10.389 24.932-24.241 0-13.851-9.696-24.238-24.932-24.238h-13.852zM643.921 205.188h53.327v11.772h-40.859v18.699H692.4v11.082h-36.012v19.392h41.553v11.08H643.92v-72.025zM713.177 205.188h13.158l22.162 33.935 22.161-33.935h13.158v72.025h-12.465v-51.249L748.497 259.9h-.692l-22.855-33.937v51.249h-11.772v-72.024zM823.293 248.818l-28.395-43.631h15.235l20.083 31.857 20.085-31.857h14.544l-28.394 43.631v29.088h-12.467v-29.088z"/></g></svg>
```

**Comment by d4l3k** [comment]
@chamkank Wasn't there another file we had issues with?

**Comment by chamkank** [comment]
Oh yeah, there was an issue where svgo would cut off the right side of the svg due to loss of precision. 
I've opened a new issue for it: #833

**Comment by GreLI** [comment]
Thanks for the report! Released v1.0.3 with the fix.
